src/ViewModel/AnnotationTeaser.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace eLife\Patterns\ViewModel;
+
+use Assert\Assertion;
+use eLife\Patterns\ArrayAccessFromProperties;
+use eLife\Patterns\ArrayFromProperties;
+use eLife\Patterns\SimplifyAssets;
+use eLife\Patterns\ViewModel;
+use Traversable;
+
+final class AnnotationTeaser implements ViewModel
+{
+    use ArrayAccessFromProperties;
+    use ArrayFromProperties;
+    use SimplifyAssets;
+
+    const RESTRICTED_ACCESS = true;
```

Comment:
What's this for?

Reply:
This is for a private annotation that is only visible to the logged in user who made it.

Reply:
Ah, see it in the tests now. A bit verbose to wrap a boolean.